### PR TITLE
Reload `postfix-setup` when mailing list membership changes

### PR DESCRIPTION
**Pull request by jfly**
This fixes https://github.com/NixOS/infra/issues/505

### CODE REVIEW

File: macs/nix-darwin.nix [hunk 1]
```diff
@@ -46,9 +46,7 @@ in
   };
 
   nix.package = pkgs.nixVersions.nix_2_24.overrideAttrs (oldAttrs: {
-    patches = oldAttrs.patches or [ ] ++ [
-      ./disable-chroot.patch
-    ];
+    patches = oldAttrs.patches or [ ] ++ [ ./disable-chroot.patch ];
```
Comment (jfly) [inline macs/nix-darwin.nix:49]: Unrelated formatting change. This snuck in in https://github.com/NixOS/infra/commit/dfd77d6dbe22ebbc2d8720d217a9947c02741723

**Review by Mic92 — APPROVED** [review]
